proto/babylon/epoching/v1/events.proto

```diff
@@ -98,3 +98,12 @@ message EventWrappedStakingUpdateParams {
   string min_commission_rate = 6;
   uint64 epoch_boundary = 7;
 }
+// EventUnlockFundsFailed is the event emmitted when unlocking funds fails.
+message EventUnlockFundsFailed {
+  uint64 epoch_number =1;
```

Comment:
weird formatting i.e there should be space after `=` , also it would be nice to doc stricng for all fields